tests/queries/0_stateless/01019_alter_materialized_view_consistent.sh

```diff
@@ -50,12 +50,19 @@ function insert_thread() {
 function alter_thread() {
     trap 'exit' INT
 
-    ALTER[0]="ALTER TABLE mv MODIFY QUERY SELECT v == 1 as test, v as case FROM src_a;"
-    ALTER[1]="ALTER TABLE mv MODIFY QUERY SELECT v == 2 as test, v as case FROM src_b;"
+    # Generate random ALTERs, but make sure that at least one of them is for each source table.
+    for i in {0..5}; do
+        ALTER[$i]="ALTER TABLE mv MODIFY QUERY SELECT v == 1 as test, v as case FROM src_a;"
+    done
+    ALTER[$RANDOM % 3]="ALTER TABLE mv MODIFY QUERY SELECT v == 2 as test, v as case FROM src_b;"
+    ALTER[$RANDOM % 6]="ALTER TABLE mv MODIFY QUERY SELECT v == 2 as test, v as case FROM src_b;"
+    ALTER[$RANDOM % 6]="ALTER TABLE mv MODIFY QUERY SELECT v == 2 as test, v as case FROM src_b;"
```

Comment:
looks like duplicate of the above line, typo?

Reply:
I wanted to insert it three times on different positions (actually, it can be `$RANDOM % 6` 3 times), I changed the first to be sure it's at the beginning of the array, but it confuses the reader. I'll add a comment or change it.